config/routes.rb

```diff
@@ -2,6 +2,14 @@
 
 Rails.application.routes.draw do
   devise_for :user, controllers: { registrations: "users/registrations" }
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+
   root to: "root#index"
+
```

Comment:
Learn what `resources` method does: https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions